js/foundation.sticky.js

```diff
@@ -215,9 +215,6 @@ class Sticky {
                   * @event Sticky#stuckto
                   */
                  .trigger(`sticky.zf.stuckto:${stickTo}`);
-    this.$element.on("transitionend webkitTransitionEnd oTransitionEnd otransitionend MSTransitionEnd", function() {
-      _this._setSizes();
-    });
```

Comment:
@gehasia is removing this technically a part of the fix for #8435 or is it just being removed in the same commit? Can we please separate these into two different commits if they're not dependent on each other? It will make tracking progress easier as we tackle Sticky's bugs. Thanks!